integration/walletextension/wallet_extension_test.go

```diff
@@ -427,6 +429,25 @@ func TestCannotSubscribeOverHTTP(t *testing.T) {
 	}
 }
 
+func TestSuccessGasEstimate(t *testing.T) {
+	createWalletExtension(t)
+	accountAddr, _ := registerPrivateKey(t)
+	callMsg := datagenerator.CreateCallMsg()
+	callMsg.From = accountAddr
+
+	callMsgBytes, err := json.Marshal(callMsg)
+	if err != nil {
+		t.Fatalf("unable to marshal CallMsg - %s", err)
+	}
+
+	callMsgHex := hexutil.Encode(callMsgBytes)
+	getBalanceJSON := makeHTTPEthJSONReqAsJSON(rpc.RPCEstimateGas, []string{callMsgHex, latestBlock})
+
+	if getBalanceJSON[walletextension.RespJSONKeyResult].(string) != "0x12a05f200" {
```

Comment:
@otherview Can we have a constant for this magic number? It would be great if it could even be calculated somehow, but I assume that's not possible?

Reply:
Good point - This will change with the next PR when the gas estimations are dynamic. Just a place holder to make sure the tests are accurate atm